packages/api/src/external/fhir-to-cda-converter/connector-lambda.ts

```diff
@@ -0,0 +1,33 @@
+import { getLambdaResultPayload, makeLambdaClient } from "@metriport/core/external/aws/lambda";
+import { Config } from "../../shared/config";
+import { FhirToCdaConverter, FhirToCdaConverterRequest } from "./connector";
+
+const region = Config.getAWSRegion();
+const lambdaClient = makeLambdaClient(region);
+const fhirToCdaConverterLambdaName = Config.getFhirToCdaConverterLambdaName();
+
+export class FhirToCdaConverterLambda implements FhirToCdaConverter {
+  async requestConvert({ cxId, patientId, bundle }: FhirToCdaConverterRequest): Promise<string[]> {
+    if (!fhirToCdaConverterLambdaName) {
+      throw new Error("FHIR to CDA Converter Lambda Name is undefined");
+    }
+
+    console.log("Invoking the lambda. We will not fail gracefully.");
```

Comment:
use log instead of console.log. Plus are the logs in here left over or intended?

Reply:
This was there for testing purposes. It has since been removed